tests/driver_vcnl40x0/main.c

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2017 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup tests
+ * @{
+ *
+ * @file
+ * @brief       Test application for the VNCL40X0 proximity and ambient light sensor.
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+#include <inttypes.h>
+
+#include "vcnl40x0.h"
+#include "vcnl40x0_params.h"
+#include "xtimer.h"
+#include "board.h"
+
+#define SLEEP_2S   (2 * 1000 * 1000u) /* 2 seconds delay between printf */
```

Comment:
why not set this to `(2U)` and use `xtimer_sleep` instead of `xtimer_usleep` below?